empathylink.lic

```diff
@@ -0,0 +1,118 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#empathylink
+=end
+
+custom_require.call(%w[common drinfomon equipmanager])
+
+class Empathylink
+  def initialize
+    if UserVars.empathylink.nil?
+      UserVars.empathylink=Hash.new
+    end
+    populate_noheal_list
+    settings = get_settings
+    @buddies = settings.hunting_buddies
+    @use_hodierna = DRC.get_settings.hashodierna
+    in_room = DRRoom.pcs
+    while true
+      in_room.each do | target |
+        if shouldheal(target)
+          touch(target)
+        end
+      end
+      pause 0.5
+      in_room = DRRoom.pcs
+    end
+  end
+
+  def populate_noheal_list
+    #UserVars.empathylink['xxxx'] = 2496928411
+    #UserVars.empathylink['xxxx'] = 2496928411
+  end
+
+  def shouldheal(target)
+    now = Time.now.to_i
+    healTime = UserVars.empathylink[target]
+    if healTime.nil? or now > healTime and Script.running?('combat-trainer')
+      pause 5
+      if DRRoom.pcs.include?(target) and  @buddies.include?(target)
+        #echo "Should heal #{target}"
+        return true
+      end
+    end
+    #echo "Shouldn't heal #{target}"
+    return false
+  end
+
+  def touch(target)
+    pause_script('combat-trainer') if Script.running?('combat-trainer')
```

Comment:
I highly recommend scoping the `unpause_script` to the same function as the `pause_script`. If I'm reading this right, `combat-trainer` could be paused here and never unpaused if we enter the case on line 62.

Reply:
All the time Ceindrech has used this the script never stayed paused. I will look into it though.

Reply:
Put unpause in same def as pause. I also made def p_script and u_script to call for the pauses - in case I still need to add them to other defs.